Initial commit (#2)
geoparquet/bigquery-converter: README.md

# bigquery-converter

Experimental python scripts to import/export [GeoParquet](https://geoparquet.org) files in [Google BigQuery](https://cloud.google.com/bigquery).

## Motivation

> BigQuery supports parquet files, but for tables with geometry data it's not supported. For example, if you try to export a table with a geography column it throws the following error:

Reviewer: ```suggestion
BigQuery supports Parquet files, but they won't work if you have geometry data. For example, if you try to export a table with a geography column it throws the following error:
```
